src/OmniSharp.Roslyn.CSharp/Services/Signatures/CheckStaticInvocation.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Sort usings. `System` namespaces go first. (Note that this is a setting in Visual Studio)